python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import absolute_import, division, print_function
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import numpy as np
+from scipy.stats import sigmaclip
+from scipy.fftpack import fft2, ifft2, ifftshift
+from scipy.ndimage.filters import convolve
+
+import lsst.afw.image as afwImage
+import lsst.meas.algorithms as measAlg
+import lsst.afw.math as afwMath
+
+__all__ = ("performExposureDecorrelation")
+
+
+def computeClippedImageStats(im, low=3, high=3):
```

Comment:
Yes. The variance plane can be treated as an ordinary image, so you can compute statistics on it just like you do on the image plane. It gets trickier if you are ignoring masked pixels.